web/docs/connectors/s3.md

```diff
@@ -42,6 +42,16 @@ that user, meaning that the AWS credentials must also be configured for that
 user. The directory `~/.aws` must be readable for the `tenzir` user.
 :::
 
+If a config file `<prefix>/etc/tenzir/plugin/s3.yaml` or
+`~/.config/tenzir/plugin/s3.yaml` exists, it is always preferred over the
+default AWS credentials. The configuration file must have the following format:
+
+```yaml
+secret-key: your-secret-key
+access-key: your-access-key
```

Comment:
The AWS documentation puts the `access-key` before the `secret-key` - maybe we should follow that order, too.
```suggestion
access-key: your-access-key
secret-key: your-secret-key
```